core/trino-main/src/main/java/io/trino/execution/CreateRoleTask.java

```diff
@@ -19,24 +19,36 @@
 import io.trino.metadata.Metadata;
```

Comment:
> Add flag to enable old catalog-only role commands

This commit should be squashed with, or go just after, the commit that introduces `IN <catalog>` syntax. We need to have tests that covers that legacy option still works as expected.

Reply:
It is too much work to move at this point.